frontend/src/lib/components/entityCard/EntityCard.svelte

```diff
@@ -34,15 +35,7 @@
     aria-posinset={ariaPosinset}
     aria-setsize={ariaSetsize}>
     <svelte:fragment slot="card-media">
-      {#if imgSrc}
-        <img class="rounded-sm" src={imgSrc} alt={imgAlt} width={imgWidth} height={imgHeight} />
-      {:else}
-        <div class="placeholder avatar">
-          <div class="w-[3.125rem] rounded-full bg-base-300">
-            <span class="text-2xl">{title.charAt(0)}</span>
-          </div>
-        </div>
-      {/if}
+      <CandidatePhoto photoURL={imgSrc} alt={imgAlt} width={imgWidth} height={imgHeight} {title} />
```

Comment:
We can remove `<svelte:fragment>` and add ` slot="card-media"` directly to `CandidatePhoto`.

If we pass the dims, they should be just `{imgWidth} {imgHeight}` but as we don't actually have those, maybe just leave them out now?